src/main/java/com/princexml/wrapper/Prince.java

```diff
@@ -353,49 +497,87 @@ public void setCssDpi(int cssDpi) {
     //endregion
 
     //region Raster output options.
+    /**
+     * Specify the format for the raster output.
+     * @param rasterFormat The format for the raster output.
+     */
     public void setRasterFormat(RasterFormat rasterFormat) {
         this.rasterFormat = rasterFormat;
     }
 
+    /**
+     * Specify the level of JPEG compression when generating raster output in JPEG format.
+     * @param rasterJpegQuality The level of JPEG compression.
+     */
     public void setRasterJpegQuality(int rasterJpegQuality) {
         if (rasterJpegQuality < 0 || rasterJpegQuality > 100) {
             throw new IllegalArgumentException("invalid rasterJpegQuality value (must be [0, 100])");
         }
         this.rasterJpegQuality = rasterJpegQuality;
     }
 
+    /**
+     * Set the page to be rasterized.
+     * @param rasterPage The page to be rasterized.
+     */
     public void setRasterPage(int rasterPage) {
         if (rasterPage < 1) {
             throw new IllegalArgumentException("invalid rasterPage value (must be > 0)");
         }
         this.rasterPage = rasterPage;
     }
 
+    /**
+     * Specify the resolution of raster output.
+     * @param rasterDpi The raster output resolution.
+     */
     public void setRasterDpi(int rasterDpi) {
         if (rasterDpi < 1) {
             throw new IllegalArgumentException("invalid rasterDpi value (must be > 0)");
         }
         this.rasterDpi = rasterDpi;
     }
 
+    /**
+     * Set the number of threads to use for multi-threaded rasterization.
+     * @param rasterThreads The number of threads to use.
```

Comment:
Do we default rasterThreads to the number of cpus or something?

Reply:
Yup. I'll mention this.